guides/source/configuring.md

```diff
@@ -1274,6 +1275,39 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `false`              |
 | 7.0                   | `true`               |
 
+#### `config.active_record.commit_transaction_on_non_local_return`
+
+Defines whether `return`, `break` and `throw` inside a `transaction` block cause the transaction to be
+committed or rolled back. e.g.:
+
+```ruby
+Model.transaction do
+  model.save
+  return
+  other_model.save # not executed
+end
+```
+
+If set to `false`, it will be rolled back.
+
+If set to `true`, the above transaction will be committed.
```

Comment:
Could we make this more explicit about what will happen? I think this is what happens, but I'm not sure:

```
If set to `false`, it will be rolled back, so neither model will save.

If set to `true`, the above transaction will be committed. `model` will save, but `other_model` will not.
```